seed/static/seed/tests/mapping_controller.spec.js

```diff
@@ -37,6 +37,11 @@ describe('controller: mapping_controller', function () {
           });
         });
 
+      spyOn(mock_geocode_service, 'check_org_has_geocoding_enabled')
+        .andCallFake(function () {
+          return $q.resolve(true);
+      });
+
```

Comment:
I wanted to point out I had to add this b/c we recently merged in some changes to the mapping_controller that didn't have the functional tests applied in CI b/c only the django tests were running on github actions at the time. This mock fixes the test.